client/post-editor/media-modal/detail/detail-item.jsx

```diff
@@ -94,17 +115,23 @@ class EditorMediaModalDetailItem extends Component {
 
 		const mimePrefix = MediaUtils.getMimePrefix( item );
 
-		if ( 'image' !== mimePrefix ) {
+		if ( ! includes( [ 'image', 'video' ], mimePrefix ) ) {
 			return null;
 		}
 
+		let editText = translate( 'Edit Image' );
+
+		if ( 'video' === mimePrefix ) {
+			editText = translate( 'Edit Thumbnail' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **79** times:
`translate( 'Thumbnail' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).